src/schema/users.ts

```diff
@@ -1743,7 +1743,9 @@ export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
 
       if (existingUserCompanyEmail) {
         if (existingUserCompanyEmail.userId !== ctx.userId) {
-          throw new ValidationError('Email already exists');
+          throw new ValidationError(
+            'Oops, this email has already been verified by another user.',
```

Comment:
do we want to give that much info in this case, it should be edge case where someone tries to verify someone else's email and its pretty suspicious case?

Reply:
Ah it's just the copy from design I can check with them?
I guess you could find out email addresses this way yeah.

Reply:
We should change it with something more generic like `Oops, this email has can't been verified at this time.`, and we can log something internally to debug it if needed.